backend/processing/model_input.py

```diff
@@ -32,5 +29,28 @@ def get_response(user_input, client):
     
     return response.choices[0].message.content
 
+def get_response_audio(text, client):
+    try:
```

Comment:
pass voice parameter to get_response_audio maybe, eventually we want to have users choose voice dyanmically

Reply:
Yeah I agree. Additionally, I think the whole body:
        audio_response = client.audio.speech.create(
            model="tts-1",
            voice="nova",
            input=text
        )
comes from a config file. Something like model = config.ttsmodel, voice = config.ttsmodel.voice, etc.